File: frontend/src/types/DTO/BusinnessTaskDTO.ts
import BusinnessTaskCommentDTO from "./BusinnessTaskCommentDTO"
import BusinnessTaskRealisationDTO from "./BusinnessTaskRealisationDTO"

type BusinnessTaskDTO = {
  name: string
  description: string
  date: Date
  term: Date
  taskRealisations: BusinnessTaskRealisationDTO[]
  comments: BusinnessTaskCommentDTO[]
}

export default BusinnessTaskDTO

File: backend/Coboss.Types/DTO/BusinnessTaskDTO.cs
using Coboss.Core.Entities;

namespace Coboss.Types.DTO
{
    public class BusinnessTaskDTO
    {
        public int Id { get; set; }
        public string Name { get; set; } = default!;
        public string Description { get; set; } = default!;
        public DateTime Date { get; set; }
        public DateTime Term { get; set; }
        public int ProjectId { get; set; } = default!;

        public ICollection<BusinnessTaskRealisationDTO> TaskRealisations { get; set; } = new List<BusinnessTaskRealisationDTO>();
        public List<BusinnessTaskCommentDTO> Comments { get; set; } = new List<BusinnessTaskCommentDTO>();
        public List<EmployeeDTO> Employees { get; set; } = new List<EmployeeDTO>();
    }
}